### PR TITLE
Setup Vet clinic DB

### PR DESCRIPTION
**Pull request by samiullahbahadur**
### **Project requirements**

-  **Create a database named vet_clinic**. 
-  **Create a table named animals with the following columns**:
- [ ] id: integer
- [ ] name: string
- [ ] date_of_birth: date
- [ ] escape_attempts: integer
- [ ] neutered: boolean
- [ ] weight_kg: decimal

- **_Insert the following data:_**

- [ ] Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg.He was neutered and he has never tried to escape.
- [ ] Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg.She is neutered and she has tried to escape 2 times.
- [ ] Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
- [ ] Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.

-  **Write queries for the following:**
- [ ] Find all animals whose name ends in "mon".
- [ ] List the name of all animals born between 2016 and 2019.
- [ ] List the name of all animals that are neutered and have less than 3 escape attempts.
- [ ] List date of birth of all animals named either "Agumon" or "Pikachu".
- [ ] List name and escape attempts of animals that weigh more than 10.5kg
- [ ] Find all animals that are neutered.
- [ ] Find all animals not named Gabumon.
- [ ] Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*Queries that provide answers to the questions from all projects.*/
+SELECT *
+FROM animals
+WHERE NAME LIKE '%mon';
+
+SELECT *
```
Comment (mikethreels) [inline queries.sql]: - [ ] this query should only list the name of the animal.

File: queries.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+FROM animals
+WHERE
+  date_of_birth BETWEEN 'January 1, 2016' AND 'December 31, 2019';
+
+SELECT *
+FROM animals
+WHERE neutered = TRUE AND escape_attempts < 3;
```
Comment (mikethreels) [inline queries.sql]: - [ ] this query should only list the name of the animal.

File: queries.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+
+SELECT date_of_birth
+FROM animals
+WHERE NAME IN ('Agumon', 'Pikachu');
+
+SELECT escape_attempts
+FROM animals
+WHERE weight_kg > 10.5;
```
Comment (mikethreels) [inline queries.sql]: - [ ] this query should only list the name of the animal.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Samiullah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (samiullahbahadur) [reply queries.sql]: Hi @mikethreels 
did you check this query or just see it 
i use like keyword and % to any name ends by mon according to requirements

File: queries.sql (hunk 2 above)
Comment (samiullahbahadur) [reply queries.sql]: Please check the  requirements

File: queries.sql (hunk 3 above)
Comment (samiullahbahadur) [reply queries.sql]: Please check the  requirements

**Review by Kingstalux — APPROVED** [review]
Hi @samiullahbahadur,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966

Good job implementing the changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**